client/signup/jetpack-connect/install-step.jsx

```diff
@@ -15,83 +16,100 @@ import JetpackExampleConnect from './example-components/jetpack-connect';
 
 const NEW_INSTRUCTIONS_JETPACK_VERSION = '4.2.0';
 
-export default React.createClass( {
-	displayName: 'JetpackInstallStep',
+class JetpackInstallStep extends Component {
 
-	confirmJetpackInstalled( event ) {
+	static propTypes = {
+		currentUrl: PropTypes.string.isRequired,
+		confirmJetpackInstallStatus: PropTypes.func.isRequired,
+		jetpackVersion: PropTypes.string,
+		onClick: PropTypes.func.isRequired,
```

Comment:
IMHO `onClick` should be optional; having it as required feels a little odd. Instead, I'd expect to see it specified as a non-required prop, but also to have a `onClick: noop` default prop value defined.